bin/MadGraph5_aMCatNLO/cards/ggh01_Toa01a01_Tomumutautau/haa_h125_10_gp/ggh01_M125_Toa01a01_M10_Tomumutautau_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+set loop_color_flows False
+set max_npoint_for_channel 0
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model NMSSMHET --modelname
+generate p p  > h01 , (h01 > a01 a01, a01 > tau+ tau-, a01 > mu+ mu-)
```

Comment:
Isn't it not "ta" instead of "tau"?

Reply:
I was following a previously accepted proc_card which used the same model
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_proc_card.dat
I could produce valid GEN/AOD files using the resulting gridpacks. My knowledge of Madgraph is limited, it is not possible/allowed to use a differnt name like this?

Reply:
I imported the model file in [1] to my standalone madgraph and the process you wrote failed because as you can see above ("all = "), your a01 particle is missing. Can you double check?

[1] https://cms-project-generators.web.cern.ch/cms-project-generators/NMSSMHET.zip

Reply:
We already discussed this offline, but just to mention that the particles are indeed defined in the model-
import model NMSSMHET --modelname
INFO: failed to load model from pickle file. Try importing UFO from File 
INFO: load particles 
INFO: load vertices 
Kept definitions of multiparticles l- / j / vl / l+ / p / vl~ unchanged
Defined multiparticle all = g u d u~ d~ a ve vm vt ve~ vm~ vt~ go su1 su2 su3 su4 su5 su6 sd1 sd2 sd3 sd4 sd5 sd6 su1~ su2~ su3~ su4~ su5~ su6~ sd1~ sd2~ sd3~ sd4~ sd5~ sd6~ c t s b c~ t~ s~ b~ z w+ h01 h02 h03 a01 a02 h+ sv1 sv2 sv3 sl1- sl2- sl3- sl4- sl5- sl6- w- h- sv1~ sv2~ sv3~ sl1+ sl2+ sl3+ sl4+ sl5+ sl6+ n1 n2 n3 n4 n5 x1+ x2+ e- mu- tau- x1- x2- e+ mu+ tau+
generate p p  > h01 , (h01 > a01 a01, a01 > tau+ tau-, a01 > mu+ mu-)

Reply:
Yes sorry I forgot to give "--modelname" option when I was running it in MadGraph.
Doing "--modelname" would use the particle names in the UFO model instead of following the MadGraph conventions. Sorry for the confusion.